oidc-handler/core/pom.xml

```diff
@@ -0,0 +1,283 @@
+            <plugin>
+                <groupId>org.apache.sling</groupId>
+                <artifactId>maven-sling-plugin</artifactId>
+                <version>2.3.4</version>
+                <configuration>
+                    <slingUrl>http://localhost:8080/system/console</slingUrl>
+                    <user>admin</user>
+                    <password>admin</password>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+    <dependencies>
+
+        <dependency>
+
+            <groupId>org.apache.felix</groupId>
+
+            <artifactId>org.apache.felix.scr</artifactId>
+
+            <version>2.0.10</version>
+
+            <scope>provided</scope>
+
+        </dependency>
+
+
+
+        <dependency>
+
+            <groupId>org.apache.felix</groupId>
+
+            <artifactId>org.apache.felix.scr.annotations</artifactId>
+
+            <version>1.9.6</version>
+
+        </dependency>
+
+        <!-- https://mvnrepository.com/artifact/com.day.cq/cq-content-sync -->
+        <!--<dependency>-->
+            <!--<groupId>com.day.cq</groupId>-->
+            <!--<artifactId>cq-content-sync</artifactId>-->
+            <!--<version>5.6.4</version>-->
+            <!--<scope>provided</scope>-->
+        <!--</dependency>-->
+
+        <dependency>
+            <groupId>org.apache.sling</groupId>
+            <artifactId>org.apache.sling.engine</artifactId>
+            <version>2.4.6</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+
+            <groupId>javax.servlet</groupId>
+            <artifactId>servlet-api</artifactId>
+            <version>2.5</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.osgi</groupId>
+            <artifactId>org.osgi.compendium</artifactId>
+            <version>4.2.0</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.osgi</groupId>
+            <artifactId>org.osgi.core</artifactId>
+            <version>4.2.0</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>javax.jcr</groupId>
+            <artifactId>jcr</artifactId>
+            <version>2.0</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.sling</groupId>
+            <artifactId>org.apache.sling.api</artifactId>
+            <version>2.9.0</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>1.7.6</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.osgi</groupId>
+            <artifactId>org.osgi.annotation.versioning</artifactId>
+            <version>1.0.0</version>
```

Comment:
You should not need this now - see https://osgi.org/javadoc/r6/annotation/index.html for details on what they do.